bedrock/firefox/views.py

```diff
@@ -23,6 +24,8 @@
 import basket
 import querystringsafe_base64
 from product_details.version_compare import Version
+from twilio.base.exceptions import TwilioRestException
```

Comment:
Is this used? I can't see TwilioRestException below, but surely Flake8 would grumble

Reply:
Yeah there's a `try / except` down below where it is used.